src/Core/Migration/MigrationHandler.php

```diff
@@ -229,9 +237,42 @@ protected function buildMigrationQueries()
         }
 
         $oDirectory = new RecursiveDirectoryIterator($this->migrationQueriesDir);
-        $oFlattened = new RecursiveIteratorIterator($oDirectory);
+        $oIterators = new AppendIterator();
+        $oIterators->append(new RecursiveIteratorIterator($oDirectory));
+
+        // Include module migration files
+        if (!self::$skipModuleMigration) {
+            $oConfig = Registry::getConfig();
+
+            if (!class_exists(ModuleList::class)) {
+                print "ERROR: Oxid ModuleList class can not be loaded,
+                please try to run vendor/bin/oe-eshop-unified_namespace_generator";
```

Comment:
Just throw an exception here, as we do it in https://github.com/OXIDprojects/oxid-console/blob/ab98dbfbe22fa5a6b63c4dfc0d7a0e36a39938c6/src/Core/Migration/MigrationHandler.php#L70

Reply:
Now throwing an exception instead of print the message.